platform/install/environments/gcp.mdx

```diff
@@ -0,0 +1,292 @@
+
+#################################################################
+
+vCluster Platform was successfully installed and can now be reached at: https://hyx4907.loft.host
+
+Thanks for using vCluster Platform!
+19:34:46 done You are successfully logged into vCluster Platform!
+- Use `vcluster platform create vcluster` to create a new virtual cluster
+- Use `vcluster platform add vcluster` to add an existing virtual cluster to a vCluster platform instance
+```
+
+When logging in via UI, provide: <Label>First Name</Label>, <Label>Last Name</Label>, <Label>Email</Label> (should default to
+the one supplied earlier) and <Label>Organization</Label>.
+
+To login via CLI, simply copy/pasted the `Login via CLI` command.
+
+This is the basic platform deployment, from there you can refer to the
+#post-install-steps to start using the platform. However there are a few
+additional configuration steps that can be done.
+</Step>
+
+</Flow>
+
+<Flow>
+
+### Expose platform UI via load balancer
+
+<Step>
+Create a LoadBalancer service to expose the platform UI.
+
+:::note
+Please note that this
+assumes the platform is deployed in the `vcluster-platform` namespace which is a
+default deployment namespace.
+:::
+
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Service
+metadata:
+  name: vcluster-platform-loadbalancer
+  namespace: vcluster-platform
+spec:
+  type: LoadBalancer
+  externalTrafficPolicy: Cluster
+  selector:
+    app: loft
+  ports:
+  - name: https
+    protocol: TCP
+    port: 443
+    targetPort: 10443
+EOF
+```
+</Step>
+
+<Step>
+Once the service is active, obtain the external IP address:
+
+```bash
+kubectl get svc vcluster-platform-loadbalancer -n vcluster-platform
+```
+
+and navigate to the IP address in your browser `https://<EXTERNAL_IP>`.
+
+:::note certificate
+Note that the platform uses a self-signed certificate, so you need to
+accept the warning in your browser. For production use, you should replace the
+certificate with a valid one.
+:::
+
+To learn more about how to configure certificate and TLS, read the [TLS
+configuration guide](/platform/configure/domain#configure-tls)
+</Step>
+
+</Flow>
+
+<Flow>
+
+### Setup custom domain and configure DNS
+
+<Step>
+Configure DNS in Google Cloud DNS.
+To use a custom domain, you need to configure the DNS to point to the LoadBalancer IP address.
+
+```bash title="Configure DNS"
+gcloud dns managed-zones create vcluster-platform --dns-name="yourdomain.tld."
+
+gcloud dns record-sets transaction start --zone=vcluster-platform
+
+gcloud dns record-sets transaction add <EXTERNAL_IP> \
+    --name="vcluster-platform.yourdomain.tld." \
+    --ttl=300 \
+    --type=A \
+    --zone=vcluster-platform
+
+gcloud dns record-sets transaction execute --zone=vcluster-platform
+```
+</Step>
+
+<Step>
+Connect the platform to the custom domain.
+
+Once the DNS is setup, the platform can be started with the command `vcluster
+platform start --host=vcluster-platform.yourdomain.tld`
```

Comment:
Isn't there also a way to configure this in either the helm chart for vcluster platform or in the platform config?